test/FunctionalTests/Microsoft.OData.Core.Tests/Json/JsonValueUtilsTests.cs

```diff
@@ -241,24 +241,38 @@ public void WriteByteShouldWork()
         {
             this.TestInit();
             var byteArray = new byte[] { 1, 2, 3 };
-            JsonValueUtils.WriteValue(this.writer, byteArray);
+            JsonValueUtils.WriteValue(this.writer, byteArray, ref this.buffer);
             this.StreamToString().Should().Be("\"AQID\"");
         }
 
+        [Fact]
+        public void WriteLongBytesShouldWork()

```

Comment:
Write edge case unit tests where size of the buffer is equal to the length of the string, etc..

Reply:
# Added.